hyprpm/src/main.cpp

```diff
@@ -24,6 +24,7 @@ constexpr std::string_view HELP = R"#(┏ hyprpm, a Hyprland Plugin Manager
 ┣ Flags:
 ┃
 ┣ --notify       | -n    → Send a hyprland notification for important events (e.g. load fail)
+┣ --notify-fail  | -nn   → Send a hyprland notification for fail events
```

Comment:
Maybe being explicit about difference between -n and -nn

Reply:
Sure, I'll add example for it too then